pdns/ws-recursor.cc

```diff
@@ -539,15 +539,23 @@ void RecursorWebServer::jsonstat(HttpRequest* req, HttpResponse *resp)
 
 void AsyncServerNewConnectionMT(void *p) {
   AsyncServer *server = (AsyncServer*)p;
+  
   try {
-    auto socket = server->accept();
+    auto socket = server->accept(); // this is actually a shared_ptr
     if (socket) {
       server->d_asyncNewConnectionCallback(socket);
     }
   } catch (NetworkError &e) {
     // we're running in a shared process/thread, so can't just terminate/abort.
+    L<<Logger::Error<<"Network error in web thread: "<<e.what()<<endl;
```

Comment:
I'm not sure we should log these at error level, even though it should in theory happen quite rarely.